plugin/src/main/java/org/opennms/plugins/cloud/tsaas/TsaasConfig.java

```diff
@@ -79,9 +93,7 @@ private Builder() {}
     private String host = "localhost";
     private int port = 5001;
     private String tokenKey = "x-scope-orgid";
```

Comment:
Same here. It should be `token` by default. Only JUnit tests can use `x-scope-orgid` or when JWT is not involved.

Reply:
fixed.